examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/LoginFragment.kt

```diff
@@ -22,25 +21,25 @@ class LoginFragment : Fragment() {
         binding.loginButton.setOnClickListener {
             binding.loginUsernameEditText.text?.toString()?.let { userId ->
                 Purchases.sharedInstance
-                    .identifyWith(
+                    .logInWith(
                         userId,
                         { error -> showUserError(requireActivity(), error) },
-                        ::advanceToOverviewFragment
+                        { _, created -> if (created) { advanceToOverviewFragment() } }
```

Comment:
this should always go to next fragment, right? Does PurchaseTester differentiate between new users and existing users that logged in?